ui/components/institutional/signature-mismatch-banner/signature-mismatch-banner.tsx

```diff
@@ -55,26 +46,12 @@ const MMISignatureMismatchBanner: React.FC = memo(() => {
     return null;
   }
 
-  return (
-    <Box
-      display={Display.Flex}
-      width={BlockSize.Full}
-      padding={4}
-      backgroundColor={BackgroundColor.primaryMuted}
-    >
-      <Icon
-        name={IconName.Info}
-        color={IconColor.infoDefault}
-        marginRight={2}
-      />
-      <Text variant={TextVariant.bodyXs} color={TextColor.textDefault}>
-        {t('mismatchAccount', [
-          shortenAddress(selectedAccount?.address),
-          shortenAddress(fromAccount?.address),
-        ])}
-      </Text>
-    </Box>
-  );
+  const message = t('mismatchAccount', [
+    shortenAddress(selectedAccount?.address),
+    shortenAddress(fromAccount?.address),
+  ]);
+
+  return <BannerAlert marginTop={3} title={message} />;
```

Comment:
```suggestion
  return <BannerAlert title={message} />;
```
hmm so this without the other change would result in
![CleanShot 2024-07-02 at 18 54 28](https://github.com/MetaMask/metamask-extension/assets/20778143/78c70aba-4d79-4735-89a3-8f13f25f321f)

which lays over a white background rather than a gray background. 

sorry, I think there might have been a misunderstanding w/ the ui/pages/confirmations/confirm/confirm.tsx file. The component can continue to be imported there as it is now, we'd just need to move the location of it 

```
            {
              ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
              <MMISignatureMismatchBanner />
              ///: END:ONLY_INCLUDE_IF
            }
            <ScrollToBottom showAdvancedDetails={showAdvancedDetails}>
```
→ 
```
            <ScrollToBottom showAdvancedDetails={showAdvancedDetails}>
              {
                ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
                <MMISignatureMismatchBanner />
                ///: END:ONLY_INCLUDE_IF
              }
```

Reply:
in case this thought occurred to anyone: usually, I'd suggest applying the margin style outside of the UI component for it to be reused with flexibility, but it appears we're only expecting to use this for a specific case and it keeps the `ui/pages/confirmations/confirm/confirm.tsx` file, which uses the component, cleaner